packages/grafana-alerting/package.json

```diff
@@ -53,17 +69,22 @@
     "react": "18.3.1",
     "react-dom": "18.3.1",
     "react-redux": "^9.2.0",
+    "rimraf": "^6.0.1",
+    "rollup": "^4.41.1",
     "type-fest": "^4.40.0",
     "typescript": "5.7.3"
   },
   "peerDependencies": {
-    "@grafana/runtime": "^12.0.0-pre",
-    "@grafana/ui": "^12.0.0-pre",
+    "@grafana/runtime": "^11.6.2",
+    "@grafana/ui": "^11.6.2",
```

Comment:
This signals that this package is incompatible with these two package versions >=12. Is that correct?

Reply:
Had to update this to get it to work with my demo plugin, but this should be compatible with both 11.x and 12.x – I'll update accordingly :)